src/views/tickets/orders.php

```diff
@@ -33,6 +33,9 @@
 $user_id   = get_current_user_id();
 $provider  = Tribe__Tickets__Tickets::get_event_ticket_provider_object( $event_id );
 
+/** @var Tribe__Tickets__Editor__Template $template */
+$template  = tribe( 'tickets.editor.template' );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: a4c9be269d11396315b33c9f8f99a916</sub>

Reply:
```suggestion
$template = tribe( 'tickets.editor.template' );
```